packages/block-editor/src/components/letter-spacing-control/index.js

```diff
@@ -15,12 +15,12 @@ import useSetting from '../../components/use-setting';
 /**
  * Control for letter-spacing.
  *
- * @param {Object}   props                      Component props.
- * @param {string}   props.value                Currently selected letter-spacing.
- * @param {Function} props.onChange             Handles change in letter-spacing selection.
- * @param {boolean}  props.__unstableInputWidth Input width to pass through to inner UnitControl.
+ * @param {Object}             props                      Component props.
+ * @param {string}             props.value                Currently selected letter-spacing.
+ * @param {Function}           props.onChange             Handles change in letter-spacing selection.
+ * @param {string|number|null} props.__unstableInputWidth Input width to pass through to inner UnitControl.
```

Comment:
While `null` is going to work at runtime (because of [this check](https://github.com/WordPress/gutenberg/blob/92662feeb220336fb28f8b79869cb251a7db4e27/packages/components/src/input-control/styles/input-control-styles.tsx#L82)), it isn't really compatible with the type used in [`InputControl`](https://github.com/WordPress/gutenberg/blob/0160b7f1ccdc806a64eb111b36b4fd9c328ad49a/packages/components/src/input-control/types.ts#L27):

To be safe, we could change this to

```suggestion
 * @param {string|number|undefined} props.__unstableInputWidth Input width to pass through to inner UnitControl.  Should be a valid CSS width value.
```

and maybe pass either `'100%'` or `''` instead of `null` ?

Reply:
The simpler approach looks good to me. The jsdoc is clear, no exceptions to linting rules, and the switch to use `100%` is explicit. Hard to misinterpret that 🙂 

Thanks! 

Let me know if you'd still prefer the jsdoc to import the CSS width property type.